tests/linearizability/linearizability_test.go

```diff
@@ -455,3 +455,11 @@ func testResultsDirectory(t *testing.T) (string, error) {
 	}
 	return path, nil
 }
+
+// forcestopCluster stops the etcd member with signal kill.
+func forcestopCluster(clus *e2e.EtcdProcessCluster) error {
```

Comment:
This could be just a method on `*e2e.EtcdProcessCluster`, but this should be good enough.